requirements.in

```diff
@@ -2,6 +2,7 @@ django
 django-debug-toolbar
 django-extensions
 django-registration
+django-rest-framework
```

Comment:
`requirements.txt` file should be generate using `pip-compile` check https://github.com/jazzband/pip-tools project.